File: scala_core/src/org/jetbrains/plugins/scala/lang/refactoring/introduceVariable/ScalaIntroduceVariableDialogInterface.java
package org.jetbrains.plugins.scala.lang.refactoring.introduceVariable;

/**
 * User: Alexander Podkhalyuzin
 * Date: 01.07.2008
 */
public interface ScalaIntroduceVariableDialogInterface {
  public boolean isOK();
  public ScalaIntroduceVariableSettings getSettings();
  public void show();
  public String getEnteredName();
  public boolean isReplaceAllOccurrences();
}

File: scala_core/src/org/jetbrains/plugins/scala/lang/refactoring/introduceVariable/ScalaIntroduceVariableDialogInterface.scala
package org.jetbrains.plugins.scala.lang.refactoring.introduceVariable

/** 
* User: Alexander Podkhalyuzin
* Date: 01.07.2008
*/

trait ScalaIntroduceVariableDialogInterface {
  def isOK(): Boolean
  def getSettings(): ScalaIntroduceVariableSettings
  def show()
  def getEnteredName(): String
  def isReplaceAllOccurrences(): Boolean
}